src/lib/icons/DetailedSpinner.tsx

```diff
@@ -0,0 +1,57 @@
+import styled, { keyframes } from 'lib/theme'
```

Comment:
nit: this spinner is statically sized to 16px, so I think it should be renamed InlineSpinner.

Reply:
Good call